feature-libs/asm/components/asm-session-timer/asm-session-timer.component.ts

```diff
@@ -20,6 +20,8 @@ export class AsmSessionTimerComponent implements OnInit, OnDestroy {
   protected interval: any;
   protected maxStartDelayInSeconds = 60000;
   timeLeft: number;
+  expiredTime: number;
+  timeout: number;
```

Comment:
It seems that this variable `timeout` is redundant, we can directly use `timeleft`, can we remove it?

Reply:
timeout is a fixed value only read from configuration in onInit, timeleft is a countdown value, also bound to html

Reply:
hmm..  I guess we can read it again to avoid store it

Reply:
updated code